frontend_lib/test/component/CardPopup.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow, configure } from 'enzyme'
+import CardPopup from '../../src/component/CardPopup/CardPopup.jsx'
+require('../../src/component/CardPopup/CardPopup.styl')
+
+describe('<CardPopup />', () => {
+  const props = {
+    customClass: 'randomCustomClass',
+    customHeaderClass: 'RandomCustomHeaderClass',
+    customColor: 'yellow',
+    onClose: () => {},
+    hideCloseBtn: false,
+    customStyle: {
+      color: 'yellow'
+    },
+    children: '<div>I am a children of CardPopup</div>'
+  }
+
+  const wrapper = shallow(
+    <CardPopup
+      customClass={props.customClass}
+      customHeaderClass={props.customHeaderClass}
+      customColor={props.customColor}
+      onClose={props.onClose}
+      hideCloseBtn={props.hideCloseBtn}
+      customStyle={props.customStyle}
+      children={props.children}
+    />
+  )
+
+  it(`should have the class "${props.customClass}"`, () =>
```

Comment:
you can try to group your tests, for example, have a describe for the design that will test the presence of class used for styling and the customStyle.
This is an example, I don't mean to do that for every component though